app/src/utils/ColorUtils.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict
+ */
+
+type rgb = [number, number, number];
+
+
+// heatmap color gradient
```

Comment:
If possible, I'd include a name of what the colormap represents so that if we want to add others later we know which is which, and so it can easily be looked up.

This one seems to match fairly closely to google's turbo colormap.